Tests/UnitTests/Networking/Backend/BackendPostReceiptDataTests.swift

```diff
@@ -984,6 +1015,17 @@ private extension BaseBackendPostReceiptDataTests {
     static let receipt = EncodedAppleReceipt.receipt("an awesome receipt".asData)
     static let receipt2 = EncodedAppleReceipt.receipt("an awesomeer receipt".asData)
     static let jws = EncodedAppleReceipt.jws("an awesomer jws token")
+    static let sk2receipt = EncodedAppleReceipt.sk2receipt(StoreKit2Receipt(
```

Comment:
Nit, but this might be clearer avoiding the repetition:
```suggestion
    static let sk2receipt = EncodedAppleReceipt.sk2receipt(.init(
```